SmartDeviceLinkTests/SDLStreamingMediaManagerSpec.m

```diff
@@ -4,13 +4,462 @@
+                OCMStub([mockAudioLifecycleManager isStreamingSupported]).andReturn(NO);
+                expect(testStreamingMediaManager.isStreamingSupported).to(beFalse());
+            });
+        });
+
+        it(@"should return the audio lifecycle manager's isAudioConnected for isAudioConnected", ^{
+            [testStreamingMediaManager isAudioConnected];
+            OCMVerify([mockAudioLifecycleManager isAudioConnected]);
+        });
+
+        it(@"should return the video lifecycle manager's isVideoConnected for isVideoConnected", ^{
+            [testStreamingMediaManager isVideoConnected];
+            OCMVerify([mockVideoLifecycleManager isVideoConnected]);
+        });
+
+        it(@"should return the audio lifecycle manager's isAudioEncrypted for isAudioEncrypted", ^{
+            [testStreamingMediaManager isAudioEncrypted];
+            OCMVerify([mockAudioLifecycleManager isAudioEncrypted]);
+        });
+
+        it(@"should return the video lifecycle manager's isVideoEncrypted for isVideoEncrypted", ^{
+            [testStreamingMediaManager isVideoEncrypted];
+            OCMVerify([mockVideoLifecycleManager isVideoEncrypted]);
+        });
+
+        it(@"should return the video lifecycle manager's isVideoStreamingPaused for isVideoStreamingPaused", ^{
+            [testStreamingMediaManager isVideoStreamingPaused];
+            OCMVerify([mockVideoLifecycleManager isVideoStreamingPaused]);
+        });
+
+        it(@"should return the video lifecycle manager's screenSize for screenSize", ^{
+            [testStreamingMediaManager screenSize];
+            OCMVerify([mockVideoLifecycleManager.videoScaleManager displayViewportResolution]);
+        });
+
+        it(@"should return the video lifecycle manager's videoFormat for videoFormat", ^{
+            [testStreamingMediaManager videoFormat];
+            OCMVerify([mockVideoLifecycleManager videoFormat]);
+        });
+
+        it(@"should return the video lifecycle manager's supportedFormats for supportedFormats", ^{
+            [testStreamingMediaManager supportedFormats];
+            OCMVerify([mockVideoLifecycleManager supportedFormats]);
+        });
+
+        it(@"should return the video lifecycle manager's pixelBufferPool for pixelBufferPool", ^{
+            [testStreamingMediaManager pixelBufferPool];
+            OCMVerify([mockVideoLifecycleManager pixelBufferPool]);
+        });
+
+        it(@"should return the video lifecycle manager's requestedEncryptionType for requestedEncryptionType", ^{
+           [testStreamingMediaManager requestedEncryptionType];
+           OCMVerify([mockVideoLifecycleManager requestedEncryptionType]);
+       });
+
+        it(@"should return the video lifecycle manager's showVideoBackgroundDisplay for showVideoBackgroundDisplay", ^{
+            [testStreamingMediaManager showVideoBackgroundDisplay];
+            OCMVerify([mockVideoLifecycleManager showVideoBackgroundDisplay]);
+        });
```

Comment:
These have slightly odd setup, but I think that they can be combined into one `it`

Reply:
I think it would be terrible to read

Reply:
Combined into three groups.